wikimedia/wikipedia-ios
# [YiR] Add Donate slide UI

**Phabricator:** https://phabricator.wikimedia.org/T378838

### Notes
This PR completes the donate slide. 

Note: This PR depends on https://github.com/wikimedia/wikipedia-ios/pull/5052. Be sure that PR is merged first.

### Test Steps
1. Fresh install app, ensure you are in the correct device region and primary app language. Ensure you are logged into an account with edits on that primary app language.
2. Go to developer settings. Enable bypass donation toggle. Go to Donate item in profile, make a donation via simulator.
3. Go to developer settings, enable YiR feature toggle. Background and foreground app.
4. Go through Year in Review. Confirm last slide looks correct (should be personalized slide, "Thank you for your contribution". Button should say "Finish" instead of "Next" and red donate button should be hidden.
5. Tap "Learn more" link in text. In-app web view should present.
6. Repeat steps 1-5, but skip step 2.
7. Now collective donate slide should appear ("0 ads..."). Red donate button should remain. Button should still say "Finish" instead of "Next.
8. Tap "Learn more" link in text. In-app web view should present, with a "Donate now" button overlay.
